practice-one/src/index.html

```diff
@@ -38,13 +35,50 @@
         <a class="btn btn-link btn-help-center" href="#">
           Help Center
         </a>
-
       </nav>
-
     </div>
-
   </header>
+  <!-- END HEADER -->
+
+  <!-- START MAIN -->
+  <main class="main-content">
+    <div class="container">
+      <!-- START SECTION GET-STARTED -->
+      <section class="section-get-started">
+        <section class="download">
+          <h1 class="heading-h1 download-heading">
```

Comment:
Naming convention for heading is not good. Please help update the class name `heading-h1`